frontend/lib/rh/rental-history.tsx

```diff
@@ -306,16 +372,37 @@ function RentalHistoryConfirmation(): JSX.Element {
       </h2>
       <p>
         <Trans id="justfix.rhWhatHappensNext">
-          You should receive your Rent History in the mail in about a week. Your
-          Rent History is an important document—it shows the registered rents in
-          your apartment since 1984. You can learn more about it and how it can
-          help you figure out if you’re being overcharged on rent at the{" "}
+          If your apartment is currently rent stabilized— or has been at any
+          point in the past— you should receive your Rent History in the mail in
+          about a week. Your Rent History is an important document—it shows the
+          registered rents in your apartment since 1984. You can learn more
+          about it and how it can help you figure out if you’re being
+          overcharged on rent at the{" "}
           <EnglishOutboundLink href="https://www.metcouncilonhousing.org/help-answers/rent-stabilization-overcharges/">
             Met Council on Housing guide to Rent Stabilization Overcharges
-          </EnglishOutboundLink>
+          </EnglishOutboundLink>{" "}
+          or by checking out our
+          <LocalizedOutboundLink
+            hrefs={{
+              en:
+                "https://www.justfix.nyc/en/learn/my-landlord-is-overcharging-me",
+              es:
+                "https://www.justfix.nyc/es/learn/my-landlord-is-overcharging-me",
+            }}
+          >
```

Comment:
Once we have more learning center links, it might be helpful for us to make a `<LearningCenterLink>` component that makes this kind of thing less verbose (it should be more DRY since we know how a learning center URL changes depending on locale).  No need to right now though!